### PR TITLE
psp-7323 Contact ambiguous

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/frontend/src/features/mapSideBar/research/tabs/fileDetails/update/UpdateSummaryForm.tsx [hunk 1]
```diff
@@ -162,11 +162,13 @@ const UpdateSummaryForm: React.FunctionComponent<IUpdateSummaryFormProps> = prop
...
           />
         </SectionField>
-        {values.requestor?.id.startsWith('P') && (
-          <SectionField label="Organization" className="pb-4">
-            {values.requestor.organizationName ?? 'none'}
-          </SectionField>
-        )}
+        {values.requestor &&
+          isPersonResult(values.requestor) &&
+          values.requestor.person?.personOrganizations?.length !== undefined && (
```
Comment (FuriousLlama) [inline source/frontend/src/features/mapSideBar/research/tabs/fileDetails/update/UpdateSummaryForm.tsx:167]: not sure this guarantees there is at least one personorganization given 0 is different than undefined

File: source/frontend/src/interfaces/IContactSearchResult.ts [hunk 2]
```diff
@@ -27,59 +19,107 @@ export interface IContactSearchResult {
   tenantType?: string;
```
Comment (FuriousLlama) [inline source/frontend/src/interfaces/IContactSearchResult.ts:19]: no need to change now, but these should not be optional, but rather allow for undefined or null instead

File: source/frontend/src/components/contact/ContactManagerView/ContactManagerView.test.tsx [hunk 3]
```diff
@@ -111,7 +110,7 @@ describe('ContactManagerView', () => {
   });
 
   it('searches by city/municipality', async () => {
-    setupMockSearch([{ ...defaultSearchResult, municipalityName: 'victoria' }]);
+    setupMockSearch([{ ...defaultPersonSearchResult, municipalityName: 'victoria' }]);
```
Comment (FuriousLlama) [inline source/frontend/src/components/contact/ContactManagerView/ContactManagerView.test.tsx:113]: we need to move away from static moc objects given some other section of the code could change it. instead, use a factory approach to have a method construct new objects on each call

**Review by FuriousLlama — APPROVED** [review]

File: source/frontend/src/components/contact/ContactManagerView/ContactManagerView.test.tsx (hunk 3 above)
Comment (devinleighsmith) [reply source/frontend/src/components/contact/ContactManagerView/ContactManagerView.test.tsx:113]: true, this isn't really part of this PR (pre-existing mock objects). This was just a re-name. I don't mind correcting this however.

**Comment by devinleighsmith** [comment]
had another look at this. There are too many issues with integrating this without the full swapover to the APIGEN models. will retarget this to 5.1 and put into draft to indicate the dependency.

**Pull request merged**